.travis.yml

```diff
@@ -14,7 +14,7 @@ branches:
 
 # Get the full commit history (required to derive our version code, see the root build.gradle)
 git:
-  depth: false
+  depth: 3
```

Comment:
As per the comment, literally **imediately above**, we require the full commit history for our Android version model, which this PR will fundamentally break.

From looking at your Github profile you've clearly done this as an automated scrape and update approach on hundreds of repos. Please don't waste our time again trying to improve your personal Github contribution statistics.